.changeset/smart-rules-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-query-builder': minor
+'@finos/legend-application': minor
+---
+
+Improve search algorithm to enhance user experience [#1294](https://github.com/finos/legend-studio/issues/1294)
```

Comment:
This message belongs to `@finos/legend-query-builder` only. Let's split this changeset and have a different message of `@finos/legend-application` to refer to virtual assistant

Reply:
Also, let's mention `property search` in this sentence